server/api/src/rsExecCmd.cpp

```diff
@@ -368,8 +371,8 @@ execCmd( execCmd_t *execCmdInp, int stdOutFd, int stdErrFd ) {
     char *av[LONG_NAME_LEN];
     int status;
 
-
-    snprintf( cmdPath, LONG_NAME_LEN, "%s/%s", CMD_DIR, execCmdInp->cmd );
+    auto cmd_path = irods::get_irods_home_directory() / CMD_DIR / execCmdInp->cmd;
```

Comment:
Is anything else using `CMD_DIR`?

If not, maybe we just delete the macro and use its value directly.

Also, `cmd_path` can be marked as const.